WiitarThing/Components/DeviceControl.xaml.cs

```diff
@@ -537,6 +537,54 @@ void device_StateChange(object sender, NintrollerStateEventArgs e)
                     holder.SetValue(Drums.InputNames.SELECT, drm.Select);
 #endregion
                 break;
+
+                case ControllerType.Turntable:
+#region Wii Turntable
+                    Turntable wtb = (Turntable)e.state;
```

Comment:
Same as with the turntable icon name:
```suggestion
                    Turntable ttb = (Turntable)e.state;
```